creator-node/src/sessionManager.js

```diff
@@ -19,26 +19,17 @@ class SessionManager {
     const tokenBuffer = await randomBytes(sessionTokenLength)
     const token = base64url.encode(tokenBuffer)
 
-    const session = await models.SessionToken.create({ cnodeUserUUID, token })
-    await redisClient.set(`SESSION.${token}`, session.cnodeUserUUID)
+    await redisClient.set(`SESSION.${token}`, cnodeUserUUID, 'EX', sessionTokenTTLSeconds)
```

Comment:
for now i think we should still keep all the session token stuff, except for verifySession. We can remove the _getSessionFromDB function from there so it effectively uses redis but we can have the reaal tokens for reference

Reply:
per discussion, tabling for now